dev-libs/botan/botan-2.6.0.recipe

```diff
@@ -60,16 +66,11 @@ BUILD_PREREQUIRES="
 	cmd:python
 	"
 
-PATCH()
-{
-	sed -i -e "/^all\>/ s/-pthread -fstack-protector/-fno-stack-protector/" src/build-data/cc/gcc.txt
-}
-
 BUILD()
 {
 	libtoolize --force --copy --install
```

Comment:
It doesn't look like they're using GNU autohell anymore. So I think we can remove this line.

Reply:
@alaviss could you push this into this PR? It looks like it's not needed any more, but my local branch is out of sync with this one :)
PS, test still fail on Entropy ... other than that it builds fine